webroot/results/admin/check_rounds.php

```diff
@@ -612,7 +612,11 @@ function checkEvents () {
       echo "  Old events list: ".implode(' ', $intersect)." <b style='color:#F00'>".implode(' ',$competitionOnly)."</b><br />\n";
       echo "  New events list: ".implode(' ', $intersect)." <b style='color:#3C3'>".implode(' ',$resultsOnly)."</b><br />\n";
       foreach($competitionOnly as $event) {
-        dbCommand("DELETE from competition_events where competition_id='$competitionId' and event_id = '$event'");
+        $competitionEventId = dbValue("SELECT id FROM competition_events WHERE competition_id='$competitionId' AND event_id = '$event'");
+        dbCommand("DELETE FROM competition_events WHERE id=$competitionEventId");
```

Comment:
Should we add an assert/sanity check that `$competitionEventId` is not null/empty?
What happens if it is in that query?

Reply:
Hi @viroulep , the `$competitionOnly` comes from https://github.com/thewca/worldcubeassociation.org/blob/84129d07c4475164bdd2f552c1e25d4abacbd922/webroot/results/admin/check_rounds.php#L586 so I don't think it needs assert.

Reply:
Ok thanks for the pointer!
I tried running the `DELETE` with an empty id and it seems the worst that can happen is a syntax error anyway.